### PR TITLE
UI layout

### PR DESCRIPTION
**Pull request by Geimaj**
This PR simplifies the UI by rendering a single token swap interface. The focus of these changes are to update the layout while restoring and simplifying all functionality from the previous layout. Another PR with design and style updates will follow

image

### CODE REVIEW

File: src/redux/actions/bridge.ts [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+            web3,
+            ethAddress!,
+            tokenData,
+          );
+        } catch (e) {
+          console.log('failed reading eth balance', e);
+        }
+
+        // do pricing request
+        const data = await priceRequestPromise;
```
Comment (musnit) [inline src/redux/actions/bridge.ts:184]: can we do this asynchronously/separately to our other queries, rather than blocking on the coingecko api?

Comment (Geimaj) [reply src/redux/actions/bridge.ts:184]: Yes absolutely, I need to address this so the code is performant with a larger set of tokens

**Review by vgeddes — APPROVED** [review]
Looks much better!

Though about the addresses displayed in the top right - surely we want to move them to the "TO" and "FROM" sections respectively? Not important for this PR, but I think its something we should address at some point.

**Comment by Geimaj** [comment]
Yes that is a good suggestion. At the moment the addresses at the top are to indicate which wallets are connected and their current gas balances. 

I added the following confirmation modal to display the addresses in the "To" and "From" sections, although I do think we could look at including this information in the main section.

image

**Comment by musnit** [comment]
those addresses are tied to the connected wallet, so may wanna keep them
there too. anyway yeh can review/qa and do any further changes in another PR

**Pull request merged**